class-lifterlms.php

```diff
@@ -98,7 +99,6 @@ private function __construct() {
 		add_action( 'init', array( $this, 'init_session' ), 6 ); // After table installation which happens at init 5.
 		add_action( 'init', array( $this, 'payment_gateways' ) );
 		add_action( 'init', array( $this, 'include_template_functions' ) );
-		add_action( 'init', array( 'LLMS_Shortcodes', 'init' ) );
```

Comment:
the core problem is that this is loaded at init 10 but too late? Couldn't we just change this to init 9 or 5 or something instead of the default 10 in that case?

Reply:
You are correct. I was concerned that changing the priority might cause problems in 3rd party code (or even our own add-ons).

Reply:
Yea you're right